### PR TITLE
CRIMAPP-542 Court of Appeal questions

### PR DESCRIPTION
**Pull request by RoseSAK**
## Description of change

Display the updated court of appeal questions

## Link to relevant ticket

[CRIMAPP-542](https://dsdmoj.atlassian.net/browse/CRIMAPP-542)

### After changes:
Screenshot 2024-04-15 at 17 36 54

## How to manually test the feature

[CRIMAPP-542]: https://dsdmoj.atlassian.net/browse/CRIMAPP-542?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: config/locales/en/casework.yml [hunk 1]
```diff
@@ -278,8 +284,6 @@ en:
       four_weeks: 4 weeks
       month: month
       annual: year
-    not_provided: Not provided
-    not_asked: Not asked when this application was submitted
```
Comment (hiboabd) [inline config/locales/en/casework.yml:282]: Should this be removed? 🤔

Comment (hiboabd) [inline config/locales/en/casework.yml:282]: I assume this was duplication?

File: spec/system/casework/viewing_an_application/application_details/case_details_spec.rb [hunk 2]
```diff
@@ -16,51 +16,96 @@
...
+        let(:application_data) do
+          super().deep_merge('case_details' => { 'appeal_original_app_submitted' => 'no' })
+        end
+
+        it 'shows the original app submitted question' do
+          expect(page).to have_content('Legal aid application for original case? No')
+        end
+
+        it 'does not show changes to financial circumstances question' do
+          expect(page).not_to have_content('Changes to financial_circumstances')
```
Comment (hiboabd) [inline spec/system/casework/viewing_an_application/application_details/case_details_spec.rb:50]: Changes to financial circumstances

File: spec/system/casework/viewing_an_application/application_details/case_details_spec.rb [hunk 3]
```diff
@@ -16,51 +16,96 @@
...
+                                               'appeal_financial_circumstances_changed' => 'yes',
                                                'appeal_with_changes_details' => 'Some details' })
       end
 
       it 'shows appeal lodged date' do
         expect(page).to have_content('Date the appeal was lodged 25/10/2021')
       end
 
+      it 'shows changes to financial circumstances question' do
+        expect(page).not_to have_content('Changes to financial_circumstances since original application? Yes')
```
Comment (hiboabd) [inline spec/system/casework/viewing_an_application/application_details/case_details_spec.rb:99]: Changes to financial circumstances

File: config/locales/en/casework.yml (hunk 1 above)
Comment (RoseSAK) [reply config/locales/en/casework.yml:282]: Yes, it was!

**Review by hiboabd — APPROVED** [review]
👍

**Pull request merged**